### DISCUSSION TITLE
Run pruned dogecoin node with docker

### DISCUSSION THREAD

**Discussion by budescode** [dogecoin/dogecoin#3630] [Help needed]
Hi, is it possible to run a pruned dogecoin node with docker?
below is my dommand

docker run --name dogecoin-node -d -v ./data:/dogecoin/data -p 44555:44555 -p 44556:44556 dogeorg/dogecoin-node:ubuntu

**Comment by patricklodder** [comment]
You should really open this issue in the repository that provides the image
